functions.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+function pullAllWorkoutsFromDb():array{
+    $connectionString = 'mysql:host=db; dbname=training_diary';
+    $dbUsername = 'root';
+    $dbpassword = 'password';
+    $db = new PDO($connectionString, $dbUsername, $dbpassword);
+
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+    $queryString = 'SELECT * FROM 
```

Comment:
Replace the * with the names of the columns you want (don't forget the backticks!).

It is not recommended to have * in production code.